akismet-api.d.ts

```diff
@@ -0,0 +1,63 @@
+declare module "akismet-api" {
+  // Type definitions for akismet-api 5.2.1
+  // Definitions by: Jason Florentino https://github.com/jasonflorentino
+
+  // Main Akismet Client Class
+  export class AkismetClient {
+    constructor(options: AkismetClientOptions);
+
+    verifyKey(callback?: () => void): void;
+
+    checkSpam(comment: Comment, callback?: () => void): boolean;
+    submitSpam(comment: Comment, callback?: () => void): void;
+    submitHam(comment: Comment, callback?: () => void): void;
```

Comment:
Is void the correct return type of these functions? They return a promise. All of the functions in this API _can_ take a callback but also return a promise for people who prefer that API. Here's the docs for the types we support: https://github.com/chrisfosterelli/akismet-api/tree/master/docs

Reply:
Ah, thanks for calling this out. I'll work on changes to ensure the typings will allow for either a a `void` return or a `Promise`. I'll try to make some time this weekend. Thanks Chris!

Reply:
Ok awesome! No rush. Thanks a bunch

Reply:
Hey Chris, feel free to take another look when you're free. I reviewed both the promise and callback usage and updated the types to properly reflect their parameters and return values.

Reply:
Awesome, thanks! Good catch on the parameters as well.